docs/content/deployment/dagster-instance.mdx

```diff
@@ -791,13 +791,3 @@ schedules:
 ```
 
 You can also set the optional `num_submit_workers` key to evaluate multiple run requests from the same schedule tick in parallel, which can help decrease latency when a single schedule tick returns many run requests.
-
-### Auto-materialize
```

Comment:
@OwenKephart I removed this because I assumed it would be axed alongside other AMP stuff. Can you confirm?

Reply:
correct, thank you!